src/resources/js/embedded-map.js

```diff
@@ -3,13 +3,14 @@
  */
 if ( "function" === typeof jQuery ) jQuery( document ).ready( function( $ ) {
 	var mapHolder,
-		position,
-		streetAddress,
-		venueCoords,
-		venueTitle;
+	    position,
+	    venueObject,
+	    venueAddress,
+	    venueCoords,
+	    venueTitle;
 
 	// The tribeEventsSingleMap object must be accessible (as it contains the venue address data etc)
-	if ("undefined" === typeof tribeEventsSingleMap ) return;
+	if ( "undefined" === typeof tribeEventsSingleMap ) return;
```

Comment:
I think just staying consistent with the file that you are in is fine and dandy. This one is mostly double quotes so sticking with those seems appropriate.

Personally, I'm lazy and prefer single quotes when I type just because it is less effort :D